api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -436,7 +436,7 @@ def __init__(
             return_sparse_X:
                 Controls whether the ``X`` data is returned as a dense or sparse :class:`torch.Tensor`. As ``X`` data is
```

Comment:
can you clarify when double-backquote vs. single-backquote is desirable?  I see double here, single elsewhere....

I assume double is emphasis (ie.., translate to a single in the final markdown), and single is intersphinx literal?

Reply:
With our current Sphinx setup:
1. Normal single backticks will render as _italic_
2. Double backticks will render as `code`
3. Any time you use references (like `:class`), you need to use single backticks.